changelog/4798.breaking.4.rst

```diff
@@ -0,0 +1,10 @@
+The keys used to format file paths in ``Fido.fetch`` have changed. They are now
+more standardised across all the clients, as they are all extracted from the
+names of the columns in the results table.
+
+
```

Comment:
```suggestion

```

Reply:
I suspect you need the double break for it to render, but who knows.

Reply:
I will leave this as unresolved to test it.